mscore/pagesettings.cpp

```diff
@@ -270,310 +325,392 @@ void PageSettings::orientationClicked()
 void PageSettings::twosidedToggled(bool flag)
       {
       preview->score()->style().set(Sid::pageTwosided, flag);
-      updateValues();
-      updatePreview(1);
+
+      for (auto w : { evenPageTopMargin, evenPageBottomMargin, evenPageLeftMargin, evenPageRightMargin })
+            w->setEnabled(flag);
+
+      evenPageLeftMargin ->blockSignals(true);
+      evenPageRightMargin->blockSignals(true);
+      if (flag) { 
+            evenPageLeftMargin ->setValue(oddPageRightMargin ->value());
+            evenPageRightMargin->setValue(oddPageLeftMargin->value());
+      }
+      else {
+            evenPageLeftMargin ->setValue(oddPageLeftMargin ->value());
+            evenPageRightMargin->setValue(oddPageRightMargin->value());
+      }
```

Comment:
This brace should be indented one more level.

Reply:
check